sinks/cache/cache_impl_test.go

```diff
@@ -37,9 +37,30 @@ func TestFuzz(t *testing.T) {
 	assert := assert.New(t)
 	assert.NoError(cache.StorePods(pods))
 	assert.NoError(cache.StoreContainers(containers))
+	time.Sleep(5 * time.Second)
+	zeroTime := time.Time{}
+	assert.NotEmpty(cache.GetPods(zeroTime, zeroTime))
```

Comment:
Did you remove the assert.NotEmpty(cache.GetFreeContainers(zeroTime, zeroTime)) line by accident?